src/EditorFeatures/VisualBasicTest/DocumentationComments/DocumentationCommentTests.vb

```diff
@@ -844,7 +844,7 @@ End Class
 ''' $$
 ''' </summary>
 Class C
-    
```

Comment:
Why is this changing?

Reply:
The commit message gives the details on this one. `CommitBufferManager` is responsible for line commit. It imports `IInlineRenameService`, which previously had no implementation in EditorFeatures so line commit was dropped from the composition. Moving the implementation down lit up this feature in an existing test.